pkgs/misc/vim-plugins/vim2nix/additional-nix-code/clang_complete

```diff
@@ -7,5 +7,8 @@
     # $ eval echo $(nix-instantiate --eval --expr 'with (import <nixpkgs>) {}; clang.default_cxx_stdlib_compile')
     preFixup = ''
       substituteInPlace "$out"/share/vim-plugins/clang_complete/plugin/clang_complete.vim \
```

Comment:
I forgot to remove `additional-nix-code` when adding the new upgrader, we only use `override.nix` these days.

Reply:
Will be removed here: https://github.com/NixOS/nixpkgs/pull/47221/files

Reply:
Thank you, I wasn't sure whether `addiotional-nix-code` was going to be required or not. I've updated the PR not to include those changes.